README.md

```diff
@@ -2,8 +2,26 @@
 
 ![License](https://img.shields.io/github/license/hackclub/burrow) ![Apple Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-apple.yml?branch=main&label=macos%2C%20ios&logo=Apple) ![Crate Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-rust.yml?branch=main&label=crate&logo=Rust)
 
-Burrow is a tool for burrowing through firewalls, built by teenagers at [Hack Club](https://hackclub.com/).
+Burrow is an open source tool for burrowing through firewalls. It is built by teenagers at [Hack Club](https://hackclub.com/).
 
-At its core, `burrow` is a command line utility written in Rust that can open virtual interfaces and direct traffic through them.
+It works through the app that allows for a quick and easy setup to open virtual interfaces and direct traffic through them. You can also use burrow directly from the command line. 
 
-You can use it directly from the command line, or you can use it wrapped inside of an app. Currently, there are apps for iOS and macOS that you can use to start the tunnel.
+## Contributing
+
+Burrow is fully open source, you can fork the repo and start contributing easily. For more information and in-depth discussions, visit the `#burrow` channel on the [Hack Club Slack](https://hackclub.com/slack/), here you can ask for help and talk with other people interested in burrow! 
+
+The project structure is divided in the following folders: 
+
+[] To do, add folder directories
+
```

Comment:
Do you want to document the git lint rules here? Or should that go in a CONTRIBUTING.md?

Reply:
We can mention it briefly, and then provide details in the CONTRIBUTING.md

Reply:
```suggestion

Burrow is fully open source, you can fork the repo and start contributing easily. For more information and in-depth discussions, visit the `#burrow` channel on the [Hack Club Slack](https://hackclub.com/slack/), here you can ask for help and talk with other people interested in burrow! For more information on how to contribute, please see [CONTRIBUTING.md]

The project structure is divided in the following folders: 

[] To do, add folder directories

```